subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/JvmProjectInitDescriptor.java

```diff
@@ -234,7 +238,7 @@ private void addTestFramework(BuildInitTestFramework testFramework, BuildScriptB
                     buildScriptBuilder
                         .plugin("Apply the groovy plugin to also add support for Groovy (needed for Spock)", "groovy")
                         .testImplementationDependency("Use the latest Groovy version for Spock testing",
-                            "org.codehaus.groovy:groovy:" + libraryVersionProvider.getVersion("groovy"));
+                            GroovyDependencyUtil.groovyModuleDependency("groovy", libraryVersionProvider.getVersion("groovy")));
```

Comment:
```suggestion
                            groovyModuleDependency("groovy", libraryVersionProvider.getVersion("groovy")));
```

Does ☝️ work as well?